tests/test_single_point.py

```diff
@@ -144,9 +146,78 @@ def test_single_point_write_nan(tmp_path):
 def test_invalid_prop():
     """Test invalid property request."""
     single_point = SinglePoint(
-        structure=DATA_PATH / "H2O.cif",
+        struct_path=DATA_PATH / "H2O.cif",
         architecture="mace",
         calc_kwargs={"model_paths": MODEL_PATH},
     )
     with pytest.raises(NotImplementedError):
         single_point.run_single_point("invalid")
+
+
+def test_atoms():
+    """Test passing ASE Atoms structure."""
+    struct = read(DATA_PATH / "NaCl.cif")
```

Comment:
Out of curiosity does ASE name its molecule sensibly you can use as a default?

Reply:
they have a collection of molecules and bulk methods.

something like bulk("NaCl", "rocksalt",cubic=True, a=5.63) will give you salt... something like molecule("H2O") will give you water... some extended one. 
we shall test both, file loaded and ase generated atoms. Elliott/Federica are both aware we travel in that direction

Reply:
There's not a way to do the reverse though, is there?

E.g. `ase.io.bulk("NaCl", "rocksalt",cubic=True, a=5.63)` gives an Atoms object `Atoms(symbols='NaClNaClNaClNaCl', pbc=True, cell=[5.63, 5.63, 5.63])`, but there's no label other than the symbols?

Reply:
nope there is not, what label are you after?

Reply:
I was just thinking it would be useful for the default structure name when we pass `Atoms` rather than the path, but I don't think it's a problem really.

Reply:
you can extract the info from Atoms.symbols that shall be enough in theory can be quite complex.

Reply:
True, yeah that's probably still better than the completely generic `"struct"`.

(I've used `get_chemical_formula` as `get_chemical_symbols` gives a list)